Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:lts-alpine
+ARG BUILD_DEPS="git g++ cmake make python2"
+WORKDIR /opt/leap-node
+RUN apk add --no-cache --update --virtual build_deps $BUILD_DEPS
+COPY . /opt/leap-node
+RUN chmod 755 bin.js
+RUN npm install --production --no-package-lock yarn
+RUN ./node_modules/.bin/yarn install --production
+RUN ./node_modules/.bin/yarn link
+RUN apk del build_deps
+CMD ["leap-node", \
+  "--config=http://node1.testnet.leapdao.org:8645", \
```

Comment:
I suggest to take this from env ARG, so we can specify which network we want connect to

Reply:
default may be `https://testnet-node1.leapdao.org` (port 46691 is opened now)